pkg/agent/sysadvisor/plugin/qosaware/qos_aware.go

```diff
@@ -92,6 +92,15 @@ func NewQoSAwarePlugin(pluginName string, conf *config.Configuration, extraConf
 		}
 	}
 
+	if !headroomReporterExists {
```

Comment:
it can also use 'resourceGetter!=nil' to determine whether it is allowed to run

Reply:
fixed